examples/just_enhancement.py

```diff
@@ -173,15 +174,6 @@ def run_scans(nScans, power_idx, DNP_data=None):
     'nPoints', 'tau_adjust_us', 'deblank_us', 'tau_us', 'nPhaseSteps', 'MWfreq', 'power_settings','pul_prog']}
 DNP_data.set_prop('acq_params',acq_params)
 # can delete this after
-print("*** *** *** *** *** *** ***")
-print("*** *** *** *** *** *** ***")
-print("*** *** *** *** *** *** ***")
-print("THIS IS MY POWER START TIMES",power_start_times)
-print("*** *** *** *** *** *** ***")
-print("*** *** *** *** *** *** ***")
-print("*** *** *** *** *** *** ***")
-for q in range(len(power_start_times)):
-    power_start_times[q] = power_start_times[q] - power_start_times[0]
```

Comment:
you don't want to do this -- just store the absolute time in the data.  You can make it relative during processing.